dev-server/src/graphql/resolvers/pokemon.js

```diff
@@ -1,156 +1,51 @@
-      const frLanguage = form.form_names.find(findByLanguage("fr"));
-      const jaLanguage = form.form_names.find(findByLanguage("roomaji"));
+  names: ({ name_en, name_fr, name_ja }) => ({
+    en: name_en,
+    fr: name_fr,
+    ja: name_ja
+  }),
 
-      formSuffix.en = enLanguage ? makeSuffix(enLanguage.name) : "";
-      formSuffix.fr = frLanguage ? makeSuffix(frLanguage.name) : "";
-      formSuffix.ja = jaLanguage ? makeSuffix(jaLanguage.name) : "";
-    }
+  types: ({ type_1_id, type_2_id }, args, { typeRepository }) =>
+    Promise.all(
+      [type_1_id, type_2_id].filter(Boolean).map(typeRepository.findTypeById)
+    ),
 
-    // For other forms, we can use the pokemon-species
-    const species = await findPokemonSpeciesByPokemonId(pokemon.id);
-    const enLanguage = species.names.find(findByLanguage("en"));
-    const frLanguage = species.names.find(findByLanguage("fr"));
-    const jaLanguage = species.names.find(findByLanguage("roomaji"));
+  evolutions: ({ id }, args, { evolutionRepository }) =>
+    evolutionRepository.findEvolutionsByPokemonId(id),
 
-    return {
-      en: enLanguage ? appendSuffix(enLanguage.name, formSuffix.en) : null,
-      fr: frLanguage ? appendSuffix(frLanguage.name, formSuffix.fr) : null,
-      ja: jaLanguage ? appendSuffix(jaLanguage.name, formSuffix.ja) : null
-    };
-  },
-  types: pokemon =>
-    pokemon.types.map(x => {
-      const typeId = extractTypeIdFromTypeUrl(x.type.url);
-
-      return findTypeById(typeId);
-    }),
-  evolutions: async pokemon => {
-    const evolutionChain = await findEvolutionChainByPokemonId(pokemon.id);
-    const evolutions = findEvolutions(pokemon, evolutionChain.chain);
-
-    return Promise.all(
-      evolutions.map(evolution => {
-        const nationalId = extractNationalIdFromSpeciesUrl(
-          evolution.species.url
-        );
+  pokedexEntries: ({ id }, args, { pokedexEntryRepository }) =>
+    pokedexEntryRepository.findPokedexEntriesByPokemonId(id),
 
-        return { pokemon: findPokemonById(nationalId) };
-      })
-    );
-  },
-  family: async pokemon => {
-    const evolutionChain = await findEvolutionChainByPokemonId(pokemon.id);
-    const nationalId = extractNationalIdFromSpeciesUrl(
-      evolutionChain.chain.species.url
-    );
+  eggGroups: ({ id }, args, { eggGroupRepository }) =>
+    eggGroupRepository.findEggGroupByPokemonId(id),
 
-    const result = await findPokemonById(nationalId);
+  abilities: ({ id }, args, { pokemonAbilityRepository }) =>
+    pokemonAbilityRepository.findByPokemonId(id),
 
-    return { pokemon: result };
-  },
-  pokedexEntries: async pokemon => {
-    const species = await findPokemonSpeciesByPokemonId(pokemon.id);
+  moves: async ({ id }, args, { moveRepository }) => {
+    const moves = await moveRepository.findMovesByPokemonId(id);
 
-    return species.flavor_text_entries.filter(x => x.language.name === "en");
+    return moves.map(x => ({
+      ...x,
+      pokemon_id: id
```

Comment:
This should probably be named `pokemonId` at this point.

Reply:
The rest of the model uses `_` so we should use the same notation I guess